### ISSUE TITLE
Windows narrator in IE not reading the combo box list items

### ISSUE THREAD

**Issue by PoojaOtari** [OfficeDev/office-ui-fabric-react#11567]
Environment Information
React fabric Package version(s): both versions 6.210.3 and 7.56.1 and tried others as well
Browser and OS versions: IE 11, Windows 10
Describe the issue:
Toggle state (on, off, changes in state) is not read by screen readers in IE 11.

Please provide a reproduction of the issue in a code pen ( IE don't support codepen):
https://developer.microsoft.com/en-us/fabric#/controls/web/combobox

To repro, open the fabric controls demo page in IE 11 and use Narrator while navigating to the combo box and try to select the items. 
Actual behavior:
Narrator won't announce the selected items in the combo box.

Expected behavior:
Narrator should announce the selected items in the combo box.

Documentation describing the expected behavior
Combobox are not accessible if the user doesn't know what item is selected
https://www.w3.org/TR/wai-aria-1.1/#combobox

**Comment by micahgodbolt** [comment]
Seems that when you open the menu via the button, focus returns to the button after the menu closes. It seems Edge has no trouble reading out the new value, but IE11 probably needs focus on the actual input

**Comment by PoojaOtari** [comment]
what is the fix though? Do I need to do any changes from our end?

**Comment by v-jashr** [comment]
Can you please provide version number of office fabric  where this solution will be available. As we haven't found  a exact solution  in comments.

**Comment by PoojaOtari** [comment]
Can you provide some update?

**Comment by jdhuntington** [comment]
@PoojaOtari please drop a line if this fix has not resolved your issue, otherwise I would like to close this issue as we believe the problem is fixed.

**Comment by micahgodbolt** [comment]
> This issue was addressed in #11570, which has now been successfully released as office-ui-fabric-react@v7.77.0
